scripts/test-standalone-core-api-server.ts

```diff
@@ -127,28 +120,30 @@ async function main(): Promise<void> {
 			DEV_WORKSPACE_FOLDER: WORKSPACE_DIR,
 			PROTOBUS_ADDRESS: `127.0.0.1:${PROTOBUS_PORT}`,
 			HOST_BRIDGE_ADDRESS: `localhost:${HOSTBRIDGE_PORT}`,
-			E2E_TEST: E2E_TEST,
-			CLINE_ENVIRONMENT: CLINE_ENVIRONMENT,
+			E2E_TEST,
+			CLINE_ENVIRONMENT,
 			CLINE_DIR: userDataDir,
 			INSTALL_DIR: extensionsDir,
 		},
 		stdio: "inherit",
 	})
+	childProcesses.push(coreService)
+
+	const shutdown = async () => {
+		console.log("\nShutting down services...")
+
+		for (const child of childProcesses) {
+			if (!child.killed) child.kill("SIGINT")
+		}
```

Comment:
We should remove from the childProcesses array here.  This would clean up the reference pointer.